packages/inter-protocol/src/clientSupport.js

```diff
@@ -0,0 +1,224 @@
+// @ts-check
+
+import { Fail } from '@agoric/assert';
+
+// XXX support other decimal places
+export const COSMOS_UNIT = 1_000_000n;
+
+// NB: not really a Proposal because the brands are not remotes
+// Instead they're copyRecord like  "{"boardId":"board0257","iface":"Alleged: IST brand"}" to pass through the boardId
+// mustMatch(harden(proposal), ProposalShape);
+/**
+ * Give/want
+ *
+ * @param {Record<string, import('@agoric/vats/tools/board-utils').BoardRemote>} brands
+ * @param {{ giveMinted?: number, wantMinted?: number } | { collateralBrandKey: string, giveCollateral?: number, wantCollateral?: number }} opts
+ * @returns {Proposal}
+ */
+const makeProposal = (brands, opts) => {
```

Comment:
Suggestion: `makeVaultProposal`

Reply:
It's any proposal tho

Reply:
I don't understand then, it seems like this uses a shape specific to vault proposals. Why do we have `makePSMProposal` still?

Reply:
derp. you're right, the keywords are different. good suggestion! force-pushing it